### PR TITLE
Fix username for AD and IDM. Fix session contexts.

### PR DESCRIPTION
**Pull request by lhellebr**

### CODE REVIEW

File: tests/foreman/ui/test_ldap_authentication.py [hunk 1]
```diff
@@ -480,13 +480,13 @@ def test_positive_update_external_user_roles(
         )
     with target_sat.ui_session(
         test_name, ldap_data['ldap_user_name'], ldap_data['ldap_user_passwd']
-    ) as session:
+    ) as ldapsession:
```
Comment (pnovotny) [inline tests/foreman/ui/test_ldap_authentication.py:483]: Probably easier would be just to remove the `session` fixture from the test signature - it's deprecated anyway - and update the line 459 above to `with target_sat.ui_session() as session:`.
Then you don't need two different variable names for each session and you won't have the name conflict with the `session` variable/fixture either.

Comment (lhellebr) [reply tests/foreman/ui/test_ldap_authentication.py:483]: But I need multiple sessions here anyway, it's intentional.
About changing deprecated `session`, I could do that but I'm currently just trying to fix failing tests to make automation greener. Not really a time for refactoring. There would surely be multiple instances of this and it's better to fix them in one commit.

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ldap_authentication.py -k test_positive_update_external_user_roles[AD]

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8767
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ldap_authentication.py -k test_positive_update_external_user_roles[AD] --external-logging
Test Result : =========== 53 deselected, 61 warnings, 1 error in 598.64s (0:09:58) ===========
```

**Comment by lhellebr** [comment]
This is waiting on config merge

File: tests/foreman/ui/test_ldap_authentication.py (hunk 1 above)
Comment (pnovotny) [reply tests/foreman/ui/test_ldap_authentication.py:483]: Ok, as a simple test fix, I can live with it. Indeed there is still plenty of occurrences of the old `session` fixture.

**Review by pnovotny — APPROVED** [review]

**Comment by lhellebr** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ldap_authentication.py -k test_positive_update_external_user_roles

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8845
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_ldap_authentication.py -k test_positive_update_external_user_roles --external-logging
Test Result : ========= 2 passed, 52 deselected, 112 warnings in 1179.95s (0:19:39) ==========
```

**Review by pondrejk — APPROVED** [review]

**Pull request merged**